docs/DEVELOPING.md

```diff
@@ -343,6 +343,114 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Microsoft SQL Connector
+
+The [Elastic Microsoft SQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Comment:
```suggestion
The [Elastic Microsoft SQL connector](../connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Use relative link instead of pointing to a specific branch.

Reply:
As per the previous discussion for `network drive` and `S3`. We point to a specific branch.

Reply:
May I know the reason? This is going to be merged to `main`, and I feel it's weird to point to a file in branch 8.7. Imagine later we create a new branch 8.8, are we going to update it to 8.8 for each link in the doc or we leave it to 8.7?

Reply:
Main reason is to not leave the branch with the connector client when clicking the link.
This was clarified in https://github.com/elastic/connectors-python/pull/549#discussion_r1126559525
and I'm good to use Chenhui's suggestion as that is more flexible as still keeps the users in the selected branch.

Reply:
Added relative link